mac

```diff
@@ -174,3 +174,11 @@ gem install rails --no-ri --no-rdoc
 if [ -f "$HOME/.laptop.local" ]; then
   . "$HOME/.laptop.local"
 fi
+
+brew cask install docker-toolbox
+
+# create the vm
+docker-machine create -d virtualbox dev
+
+# import environment variables for the docker-cli
+eval "$(docker-machine env dev)"
```

Comment:
this last bit not needed here. Actually it should be added in '.bashrc' and `.zshrc` as it needs to be reload in each terminal terminal window